packages/sit-onyx/src/components/OnyxFormElement/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { FormErrorMessages } from "../../composables/useCustomValidity";
+import type { RequiredMarkerProp } from "../../composables/required";
+
+export type OnyxFormElementProps = RequiredMarkerProp & {
+  /**
+   * Label to show above the form element.
+   */
+  label?: string;
```

Comment:
Label should be required for aria reasons, we offer a `hideLabel` property usually to hide it visually

Reply:
ok 👍